sections/06_discussion.md

```diff
@@ -210,6 +210,196 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
+and drug toxicity prediction
+[@tag:Mayr2016_deep_tox @tag:Hughes2016_macromol_react]. Kearnes et al.
+[@tag:Kearnes2016_admet] did a systematic comparison of single-task and
+multi-task deep models on a set of industrial ADMET datasets. They confirmed
+that multi-task learning can improve performance over single-task models. They
+further showed that smaller datasets tend to benefit more from multitask
+learning than larger datasets. Results emphasized that multitask effects are
+highly dataset-dependent, suggesting the use of dataset-specific models to
+maximize overall performance.
+
+MTL approach is complementary to multimodal and transfer learning. All three
+techniques can be used together in the same model. For example, Zhang et al.
+[@tag:Zhang2015_multitask_tl] combined deep model based transfer and multi-task
+learning for cross-domain image annotation. One could imaging extending that
+approach to multimodal inputs as well. Common characteristic os these methods
+lies in better generalization of extracted features by leveraging relationships
+between information in provided in inputs and task objectives, represented at
+various hierarchical levels of abstraction in a deep learning model structure.
+
+Despite demonstrated improvements, transfer learning approaches also pose
+a number of challenges. As mentioned above, there are no theoretically sound
+principles for pre-training and fine-tuning. Most best practice recommendations
+are heuristic and have to take into account additional hyper-parameters that
+depend on specific deep architectures, sizes of pre-training and target
+datasets, and similarity of domains. However, similarity of datasets and domains
+in transfer learning and relatedness of tasks in MTL are difficult to access.
+Most current studies address these limitations by empirical evaluation of the
+model using established best practices or heuristics and cross-validation.
+Unfortunately, negative results are typically left out and not
+presented in the final study publications.
+Results by Rajkomar et al. [@tag:Rajkomar2017_radiographs] showed that a deep
+CNN trained on natural images can boost radiology image classification
+performance. However, due to differences in imaging domains, target task required
+either re-training the initial model from scratch with special pre-processing
+or fine-tuning of the whole network on radiographs with heavy data augmentation
+to avoid overfitting. Exclusively fine-tuning top layers led to much lower
+validation accuracy (81.38 vs 99.5).
+Fine-tuning procedure for the discussed Basset model pre-trained on data
+from different cell types required no more than one training pass. Otherwise,
+the model started overfitting new data [@tag:Kelley2016_basset].
+DeepChem framework successfully improved results for
+low-data drug discovery with one-shot learning, but it was not able to achieve
+any predictive power with transfer learning on closely related testing tasks
```

Comment:
I'm not sure that "closely related testing tasks from different datasets" is the correct characterization.  Their one shot learning was unsuccessful when crossing between Tox21 and SIDER.  Those are rather different types of datasets.  It may be more accurate to say something like:

> DeepChem successfully improved results for low-data drug discovery with one-shot learning for related tasks.  However, it demonstrated clear limitations to cross-task generalization across unrelated tasks, specifically nuclear receptor assays and patient adverse reactions  [@tag:AltaeTran2016_one_shot].